2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_setup.py:_flush():68] Configure stats pid to 220593
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_setup.py:_flush():68] Loading settings from /home/peppe/.config/wandb/settings
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_setup.py:_flush():68] Loading settings from /home/peppe/Desktop/Università/Projects/RL_Project-Generalising_game_environments_with_curiosity-driven_agents/wandb/settings
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_init.py:_log_setup():476] Logging user logs to /home/peppe/Desktop/Università/Projects/RL_Project-Generalising_game_environments_with_curiosity-driven_agents/wandb/run-20221220_105258-i4tlkh8c/logs/debug.log
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_init.py:_log_setup():477] Logging internal logs to /home/peppe/Desktop/Università/Projects/RL_Project-Generalising_game_environments_with_curiosity-driven_agents/wandb/run-20221220_105258-i4tlkh8c/logs/debug-internal.log
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f4b90a7f880>
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_init.py:init():516] calling init triggers
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_init.py:init():569] starting backend
2022-12-20 10:52:58,549 INFO    MainThread:220593 [wandb_init.py:init():573] setting up manager
2022-12-20 10:52:58,551 INFO    MainThread:220593 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-12-20 10:52:58,552 INFO    MainThread:220593 [wandb_init.py:init():580] backend started and connected
2022-12-20 10:52:58,563 INFO    MainThread:220593 [wandb_run.py:_label_probe_notebook():1116] probe notebook
2022-12-20 10:52:58,563 INFO    MainThread:220593 [wandb_run.py:_label_probe_notebook():1126] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-12-20 10:52:58,563 INFO    MainThread:220593 [wandb_init.py:init():658] updated telemetry
2022-12-20 10:52:58,567 INFO    MainThread:220593 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-12-20 10:52:58,990 INFO    MainThread:220593 [wandb_run.py:_on_init():2006] communicating current version
2022-12-20 10:52:59,100 INFO    MainThread:220593 [wandb_run.py:_on_init():2010] got version response 
2022-12-20 10:52:59,101 INFO    MainThread:220593 [wandb_init.py:init():728] starting run threads in backend
2022-12-20 10:53:01,515 INFO    MainThread:220593 [wandb_run.py:_console_start():1986] atexit reg
2022-12-20 10:53:01,515 INFO    MainThread:220593 [wandb_run.py:_redirect():1844] redirect: SettingsConsole.WRAP_RAW
2022-12-20 10:53:01,515 INFO    MainThread:220593 [wandb_run.py:_redirect():1909] Wrapping output streams.
2022-12-20 10:53:01,515 INFO    MainThread:220593 [wandb_run.py:_redirect():1931] Redirects installed.
2022-12-20 10:53:01,515 INFO    MainThread:220593 [wandb_init.py:init():765] run started, returning control to user process
2022-12-20 10:53:01,517 INFO    MainThread:220593 [wandb_init.py:_pause_backend():389] pausing backend
2022-12-20 10:53:01,517 INFO    MainThread:220593 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-12-20 10:53:11,210 INFO    MainThread:220593 [wandb_init.py:_resume_backend():398] resuming backend
2022-12-20 10:56:35,635 INFO    MainThread:220593 [wandb_init.py:_pause_backend():389] pausing backend
2022-12-20 10:56:35,636 INFO    MainThread:220593 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-12-20 12:01:11,899 WARNING MsgRouterThr:220593 [router.py:message_loop():77] message_loop has been closed
